apps/src/templates/levelSummary/CheckForUnderstanding.jsx

```diff
@@ -46,7 +46,7 @@ const CheckForUnderstanding = ({
         {nextLevel && (
           <a
             className={isRtl ? styles.navLinkLeft : styles.navLinkRight}
-            href={`${nextLevel.url}${document.location.search}`}
+            href={nextLevel.url}
```

Comment:
not technically related to this pr, but necessary to go out with it, since we don't want people navigating around to broken level summary pages